pages/Registration/Login/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { Box, ToggleButton, ToggleButtonGroup } from "@mui/material";
+
+import { RegistrationLayout } from "@components/Registration";
+import { SignUpForm } from "@components/Registration/signup";
+import { LoginForm } from "@components/Registration/login";
+
+export const Login = () => {
+  const [page, setPage] = useState("signIn");
+  const handleChange = (event, newPage) => {
```

Comment:
`newPage` would be undefined in all cases unless passed explicitly
```javascript
onChange={(e) => handleChange(e, 'any extra param i-e newPage')}
```

Reply:
You are correct thats why then next line is added as default Fallback in case of undefined!